components/sections/CompanyOverview/index.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from "react";
+import ArrowIcon from "../../ui/ArrowIcon";
+import Overview from "./Overview";
+import { motion as Motion } from "framer-motion";
+import companyTabs from "./../../../data/companyTabs";
```

Comment:
As data is in the root directory, you should access it by `/`

```suggestion
import companyTabs from "/data/companyTabs";
```